roles/manifest/README.md

```diff
@@ -0,0 +1,69 @@
+theforeman.foreman.manifest
+===============================================
+
+Upload Subscription Manifest to Katello
+
+Requirements
+------------
+
+- `theforeman.forman` collection
+
+Role Variables
+--------------
+
+- `server_url`: URL of the Foreman server.
+- `username`: Username accessing the Foreman server.
+- `password`: Password of the user accessing the Foreman server.
+- `validate_certs`: Whether or not to verify the TLS certificates of the Foreman server.
+- `organization`: Organization to upload Manifest to.
+- `manifest_path`: Where to find the Manifest on Foreman server. When using download_manifest, it is first downloaded to this location.
+- `download_manifest`: Whether to first download the Manifest from the Red Hat Customer Portal.
```

Comment:
I think this being specific to the role would be `manifest_download_manifest` or `manifest_download`